googleapis/google-cloud-node
# migrate code from googleapis/nodejs-web-security-scanner

- chore: add initial files for launch
- fix header
- chore: add initial files for launch
- feat!: initial stub of library
- feat!: write initial samples (#2)
- chore: release 1.0.0 (#1)
- build: correct package name for npm (#10)
- chore: update flakybot config and sample index (#11)
- build: update gapic-generator-typescript to v1.2.10. (#12)
- chore: migrate to owl bot (#13)
- chore(deps): update dependency sinon to v10 (#18)
- chore(deps): update dependency @types/sinon to v10 (#30)
- chore(deps): update dependency ts-loader to v9 (#34)
- fix(deps): require google-gax v2.12.0 (#38)
- chore: new owl bot post processor docker image (#40)
- fix: use require() to load JSON protos (#41)
- chore: update gapic-generator-typescript to v1.3.2 (#42)
- chore: release 1.0.1 (#39)
- chore: new owl bot post processor docker image (#44)
- chore(deps): update dependency sinon to v11 (#45)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#46)
- chore: release 1.0.2 (#47)
- chore: make generate_index_ts() deterministic (#49)
- fix: make request optional in all cases (#55)
- chore: release 1.0.3 (#56)
- fix(deps): google-gax v2.17.0 with mTLS (#59)
- chore: release 1.0.4 (#60)
- fix(deps): google-gax v2.17.1 (#62)
- chore: release 1.0.5 (#63)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#64)
- chore: release 1.0.6 (#65)
- chore(nodejs): update client ref docs link in metadata (#70)
- fix(deps): google-gax v2.24.1 (#72)
- chore: release 1.0.7 (#73)
- feat: turns on self-signed JWT feature flag (#74)
- chore: release 1.1.0 (#75)
- chore: disable renovate dependency dashboard (#1194) (#76)
- fix(build): migrate to using main branch (#77)
- chore: release 1.1.1 (#78)
- build(node): update deps used during postprocessing (#1243) (#85)
- chore(cloud-rad): delete api-extractor config (#86)
- chore: fix the wrong post processor image (#88)
- chore(deps): update dependency @types/node to v16 (#87)
- chore(deps): update dependency sinon to v12 (#89)
- docs(samples): add example tags to generated samples (#90)
- build: add generated samples to .eslintignore (#91)
- docs(node): support "stable"/"preview" release level (#1312) (#94)
- chore: add api_shortname and library_type to repo metadata (#93)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#96)
- test(nodejs): remove 15 add 16 (#1322) (#98)
- build: update copyright year to 2022 (#100)
- chore: update v2.12.0 gapic-generator-typescript (#102)
- chore(deps): update dependency sinon to v13 (#104)
- docs(nodejs): version support policy edits (#1346) (#106)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#107)
- chore: update v2.14.2 gapic-generator-typescript (#111)
- build(node): update client library version in samples metadata (#1356) (#118)
- chore(deps): update dependency @types/mocha to v9 (#119)
- chore(deps): update dependency sinon to v14 (#122)
- build!: update library to use Node 12 (#124)
- chore(main): release 2.0.0 (#125)
- chore(deps): update dependency jsdoc-fresh to v2 (#127)
- chore(deps): update dependency jsdoc-region-tag to v2 (#128)
- feat: support regapic LRO (#129)
- chore(main): release 2.1.0 (#130)
- chore(deps): update dependency linkinator to v4 (#134)
- fix: better support for fallback mode (#135)
- fix: change import long to require (#136)
- chore(main): release 2.1.1 (#137)
- fix: do not import the whole google-gax from proto JS (#1553) (#139)
- chore(main): release 2.1.2 (#141)
- fix: preserve default values in x-goog-request-params header (#140)
- test: use fully qualified request type name in tests (#144)
- fix(deps): use google-gax v3.5.2 (#148)
- chore(deps): update dependency @types/node to v18 (#146)
- chore(deps): update dependency jsdoc to v4 (#150)
- fix: regenerated protos JS and TS definitions (#151)
- chore(main): release 2.1.3 (#142)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
